test/tools/router_check/test/route_tests.sh

```diff
@@ -26,8 +26,8 @@ BAD_CONFIG_OUTPUT=$(("${PATH_BIN}" "${PATH_CONFIG}/Redirect.golden.json" "${PATH
 
 # Failure test case
 echo testing failure test case
-FAILURE_OUTPUT=$("${PATH_BIN}" "${PATH_CONFIG}/TestRoutes.yaml" "${PATH_CONFIG}/Weighted.golden.json" "--details") ||
+FAILURE_OUTPUT=$("${PATH_BIN}" "${PATH_CONFIG}/TestRoutes.yaml" "${PATH_CONFIG}/Weighted.golden.json" "--details" 2>&1) ||
 echo ${FAILURE_OUTPUT}
```

Comment:
What do you think of making this `echo "${FAILURE_OUTPUT:-no output}"`? That will produce a line that says "no output" should that happen.

This line should also be indented two, since it's a continuation of the previous line.

Reply:
Sounds reasonable to me